Workiva/frugal
# CT-29  Share packing and unpacking code in dart generated files

### Story:
The dart generated code is very verbose, more so than the go version. In go, prepareMessage and processReply are shared and generic. Update the dart generation to reflect that behavior. 

### Acceptance Criteria:

### Design Notes:
Pull repeated code out of the dart functions and into a shared member. Reduce the total file size of each service provider. 

### How To Test:
I ran this against DPC here to verify the behavior against many services. https://github.com/Workiva/doc_plat_client/pull/12635.

With the latest changes, this is less straight forward and requires the dart dep to be upgraded before you can use it. As a result the PR no longer passes CI. 

### My Test Results:
This reduced the total line count in DPC by ~4000 lines and reduced the generated js size. I would expect the same to apply on a grander scale to all of workiva's dart code. 

#### Reviewers:
@Workiva/product2
